shell/Dockerfile

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
-FROM hyperledger/sawtooth-shell:nightly
+# Change 'nightly' to 'bumper-nightly' when you are using Sawtooth 1.1/Ubuntu 18 LTS (Xenial)
```

Comment:
Ubuntu 18 is Bionic, not Xenial (Xenial is Ubuntu 16).
And `nightly`  should work now with Ubuntu 18 (Bionic).

Reply:
I am running it on mac getting the same error.